codingTest_김준우/완주하지 못한 선수.js

```diff
@@ -0,0 +1,14 @@
+function solution(participant, completion) {
+  let answer = "";
+  participant = participant.sort();
+  completion = completion.sort();
+
+  completion.push("");
+  
+  for(let i = 0; i<participant.length; i++){
+         if(participant[i] !== completion[i]){
+             answer = participant[i];
+             break;
+         }
+      } return answer;
```

Comment:
```suggestion
  for(let i = 0; i<participant.length; i++){
         if(participant[i] !== completion[i]){
             return participant[i];
         }
      } 
```
어짜피 ```for문```을 순회하다가 ```if문```을 만나면 ```answer```에 담고  ```break```후 바로 ```return```하고 있기 때문에 그 과정을 생략시켜도 괜찮을 것 같습니다. ```return```을 만나는 순간 함수가 종료되는걸 활용한 것 입니다. 이렇게 하면 위에서 ```let answer = "";```도 선언할 필요가 없어지겠죠?